tardis/model/base.py

```diff
@@ -72,7 +72,14 @@ def __init__(self, velocity, homologous_density, abundance, time_explosion,
         self.v_boundary_outer = v_boundary_outer
         self.homologous_density = homologous_density
         self._abundance = abundance
+        self.decayed = False
         self.time_explosion = time_explosion
+
+        self.raw_abundance = self._abundance
```

Comment:
We should discuss and define at some point the names of all `abundance` variables involved.
There is input: `abundance` and `isotope_abundance` (should be attached to model for diagnosis and debugging purposes)
There is output: `Model.abundance` which should only contain the region defined by the velocity boundaries (as it currently does) AND which I think should contain the merged result if `isotope_abundance` is passed as an optional argument.
Then we have `self._abundance` which looks like it's currently the reference to the input (for diagnosis)

So what we basically should decide is, how do we cache the result of the decay (do we even cache it?) and how do we attach the input to the model.